deps/chakrashim/core/CONTRIBUTING.md

```diff
@@ -0,0 +1,54 @@
+## Contributing Code
```

Comment:
> Why not use a git submodule?
This follows the convention of existing deps -- source copy instead of submodule. It allows more flexibility such as making changes here before upstream when necessary (Not familiar with submodule, don't know if that's also supported).